ui/src/components/PostModal.tsx

```diff
@@ -43,7 +49,40 @@ export const PostModal: React.FC<Props> = ({ post, isModalOpen, setIsModalOpen }
                 skills={post.skillsPossessed}
                 className="[--skill-color:theme(colors.accent2)]"
             />
-            <p>{post.description}</p>
+            <p className="mb-16">{post.description}</p>
+
+            <ModalCallToAction authorName={post.author} authorId={post.authorId} />
         </Modal>
     )
+}
+
+/**
+ * Present Discord CTA to user
+ *
+ * The direct link has been a bit finicky in the past - we should make this more robust where possible
+ * TODO: Investigate if app links are feasible
+ * TODO: Don't display if user fails Guild Permissions check
+ */
+const ModalCallToAction: React.FC<CTAProps> = ({ authorName, authorId }) => {
```

Comment:
Nit: confusing name. It seemed like this was going to open a model of its own, rather than just being contained within one.

Reply:
What would be a good naming convention to use here?

Reply:
Maybe `MessageOnDiscordButton`?

Reply:
Updated 👍